src/components/portrait/portrait.test.js

```diff
@@ -0,0 +1,281 @@
+import React from "react";
+import Portrait from "./portrait.component";
+import Box from "../box";
+import CypressMountWithProviders from "../../../cypress/support/component-helper/cypress-mount";
+import { icon, getDataElementByValue } from "../../../cypress/locators";
+
+import {
+  portraitPreview,
+  portraitInitials,
+  portraitAlt,
+} from "../../../cypress/locators/portrait";
+
+const testData = ["mp150ú¿¡üßä", "!@#$%^*()_+-=~[];:.,?{}&\"'<>"];
+const testImage = [
+  "https://www.gravatar.com/avatar/cceff1ad774bf89b1f75cb6e5005333c?s=40&d=404",
+];
+
+const PortraitComponent = ({ ...props }) => {
+  return <Portrait {...props} />;
+};
+
+const TooltipPortraitComponent = ({ ...props }) => {
+  return (
+    <Portrait
+      tooltipMessage="Rebecca Smith"
+      tooltipIsVisible
+      src="https://avataaars.io/?avatarStyle=Transparent&topType=LongHairStraight&accessoriesType=Blank&hairColor=BrownDark&facialHairType=Blank&clotheType=BlazerShirt&eyeType=Default&eyebrowType=Default&mouthType=Default&skinColor=Light"
+      {...props}
+    />
+  );
+};
+
+context("Tests for Portrait component", () => {
+  describe("check props for Portrait component", () => {
+    it.each([
```

Comment:
Why have you used an array for just one instance? You could just set the `emailID` and `srcImage` as constants.

Reply:
Fixed it